custom/icds_reports/management/commands/rebuild_ccs_record_ucr.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import, print_function
+from __future__ import unicode_literals
+
+import time
+
+from django.core.management.base import BaseCommand
+from corehq.apps.userreports.tasks import _get_config_by_id, _build_indicators
+from corehq.apps.change_feed.data_sources import get_document_store_for_doc_type
+from custom.icds_reports.models.aggregate import CcsRecordMonthly
+from pillowtop.dao.couch import ID_CHUNK_SIZE
+
+
+class Command(BaseCommand):
+    help = "Rebuild ccs record cases ucr for pnc_complete"
+
+    def handle(self):
+        indicator_config_id = 'static-ccs_record_cases'
+        case_type_of_xmlns = 'ccs_record'
+        config = _get_config_by_id(indicator_config_id)
+        document_store = get_document_store_for_doc_type(
+            config.domain, config.referenced_doc_type, case_type_or_xmlns=case_type_of_xmlns
+        )
+
+        doc_ids = CcsRecordMonthly.objects.filter(pnc_complete=1).values('case_id')
```

Comment:
This has one row for every ccs_record case for every month, which will be millions of more rows than necessary. It would be much faster to add a month filter for the current month.

Reply:
added the month filter